src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 5px;
+    color: #060b35;
+    line-height: 18px;
+  }
+  &__code {
+    font-size: 10px;
```

Comment:
Font sizes should be defined using `rem` units instead of `px` to ensure better scalability and accessibility. Consider converting '10px' to the equivalent 'rem' value.